src/sentry/api/endpoints/team_issue_breakdown.py

```diff
@@ -0,0 +1,54 @@
+import copy
+from datetime import timedelta
+
+from django.db.models import Count
+from django.db.models.functions import TruncDay
+from rest_framework.response import Response
+
+from sentry.api.base import EnvironmentMixin
+from sentry.api.bases.team import TeamEndpoint
+from sentry.api.utils import get_date_range_from_params
+from sentry.models import GroupHistory, GroupHistoryStatus, Project
+from sentry.models.grouphistory import ACTIONED_STATUSES
+
+
+class TeamIssueBreakdownEndpoint(TeamEndpoint, EnvironmentMixin):
+    def get(self, request, team):
+        """
+        Returns a dict of team projects, and a time-series dict of issue stat breakdowns for each.
+
+        Right now the stats we return are the count of reviewed issues and the total count of issues.
+        """
+        project_list = Project.objects.get_for_team_ids(team_ids=[team.id])
+        start, end = get_date_range_from_params(request.GET)
+        end = end.date() + timedelta(days=1)
+        start = start.date() + timedelta(days=1)
+        bucketed_issues = (
+            GroupHistory.objects.filter(
+                organization_id=team.organization_id,
```

Comment:
It is probably better to not filter on `organization_id` here, since we don't have an index on this, and the project ids should be specific enough.